### ISSUE TITLE
Python2 compatible

### ISSUE THREAD

**Issue by arvindkhadri** [Overseas-Student-Living/sqlalchemy-filters#18]
Hi, 
I am trying to use this project on python2 and I get the following error,
```python
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/sqlalchemy_filters/__init__.py", line 3, in <module>
    from .filters import apply_filters  # noqa: F401
  File "/usr/lib/python2.7/site-packages/sqlalchemy_filters/filters.py", line 3, in <module>
    from inspect import signature
ImportError: cannot import name signature
```

**Comment by mattbennett** [comment]
We chose not to support Python 2 for this library, sorry.

`inspect.signature` is not available before Python 3.3. You may be able to work around it in a fork if you can't upgrade to Python 3.

**Comment by arvindkhadri** [comment]
I have a work around, I was wondering if I should send a pull request.
